http/http.go

```diff
@@ -145,9 +151,10 @@ func (s *Server) newResponse(r *http.Request) (Response, error) {
 		ASN:        autonomousSystemNumber,
 		ASNOrg:     asn.AutonomousSystemOrganization,
 		Hostname:   hostname,
-		UserAgent:  userAgent,
 	}
 	s.cache.Set(ip, response)
+	userAgent := userAgentFromRequest(r)
+	response.UserAgent = userAgent
```

Comment:
Nice idea moving the lines down 👍 
Can even join them down to a single line, `response.UserAgent = userAgentFromRequest(r)`

Reply:
:+1: